protocols/kad/src/behaviour.rs

```diff
@@ -624,19 +624,20 @@ where
     pub fn remove_peer(
         &mut self,
         peer: &PeerId,
-    ) -> Option<kbucket::EntryView<kbucket::Key<PeerId>, Addresses>> {
-        let key = kbucket::Key::from(*peer);
+    ) -> Option<kbucket_priv::EntryView<kbucket_priv::Key<PeerId>, Addresses>> {
```

Comment:
So this is one example of a public API that returns a now "private" type.

We need to export `EntryView` at the root to allow users to reference this type without deprecations.

Can you scan for others as well please?

Reply:
@thomaseizinger Done, just didn't export these ones (which are only used inside functions and therefore not intended to be public as far as I understand):
-  `kbucket_priv::Entry`
- `kbucket_priv::InsertResult`
- `kbucket_priv::Distance`
- `kbucket_priv::Node`

Reply:
Sounds good, thank you!

It is not a drama if we missed something as they are still accessible via the deprecated modules so a users can open an issue and tell us :)